src/implementations/deploy/token-registry/token-registry.ts

```diff
@@ -0,0 +1,33 @@
+import { readFileSync } from "fs";
+import { TradeTrustERC721Factory } from "@govtechsg/token-registry";
+import { TradeTrustERC721 } from "@govtechsg/token-registry/types/TradeTrustERC721";
+import { Wallet, getDefaultProvider } from "ethers";
+
+interface DeployTokenRegistryCmd {
+  registryName: string;
+  registrySymbol: string;
+  network: string;
+  key?: string;
+  keyFile?: string;
+}
+
+export const getKeyFromFile = (file?: string): undefined | string => {
+  return file ? readFileSync(file).toString() : undefined;
+};
+
+export const deployTokenRegistry = async ({
+  registryName,
+  registrySymbol,
+  network,
+  key,
+  keyFile
+}: DeployTokenRegistryCmd): Promise<TradeTrustERC721> => {
+  const privateKey = key || getKeyFromFile(keyFile) || process.env["OA_PRIVATE_KEY"];
+  if (!privateKey)
+    throw new Error("No private key found in OA_PRIVATE_KEY, key or key-file, please supply at least one");
+  const provider = getDefaultProvider(network === "mainnet" ? undefined : network); // undefined network defaults to homestead (aka mainnet)
```

Comment:
why not using homestead directly ? (I think I had the question for identity wallet where we perform the same conversion)

Reply:
i remembered someone saying homestead sounds foreign to people and should use mainnet instead lol

Reply:
ok for keeping mainnet, is it better to change undefined to "homestead ?